### ISSUE TITLE
Branded install should kick off login prompt after installation completes

### ISSUE THREAD

**Issue by MTRichards** [owncloud/client#2752]
Challenge with not auto-initiating connection assistant after installing ownCloud client - in this case, in a branded client with URL pre-configred.

Expectation: Once the client is installed, the connection wizard pops up and starts the connection process for the user.

What happens:
For a fresh install, once the client is installed, it does not go directly to the connection assistant, prompting for the loggin/password but instead the user has to fins the icon in the task bar, reopen the GUI, Go to the update account menu and finally gets access to the connection assistant. I know our users will never go that far. This is particularly problematic since once the install is done, the user expects everything to be working and that the synchronization will start automatically while instead, the client just sits doing nothing. I know it’s not customization but rather development but it would be really helpful to us since we are planning to deploy the client with a script and our end-users will therefore have not direct hotline assistance.

**Comment by danimo** [comment]
Which platform?

**Comment by MTRichards** [comment]
@rperezb ?

**Comment by rperezb** [comment]
Checking, in the meantime, I have reproduced it on MAC

**Comment by rperezb** [comment]
Checked, both Windows and Mac; being more important on Windows. thx

**Comment by danimo** [comment]
@rperezb On Windows, it should just work like that: Once the installer is done, the app starts up with a wizard.

OS X used to do that, but with the new installer this no longer the case.

**Comment by danimo** [comment]
In that case, this issue would be a dupe of https://github.com/owncloud/client/issues/2410

**Comment by ogoffart** [comment]
Closing as duplicate of  #2410
